superbit_lensing/galsim/mock_superBIT_gaussian.py

```diff
@@ -814,6 +826,31 @@ def main():
         wcs = galsim.TanWCS(affine, sky_center, units=galsim.arcsec)
         full_image.wcs = wcs
 
+        ##
+        ## Check for the existence of star, galaxy and cluster galaxy seeds
+        ## If they don't exist, create new ones
+
+        if sbparams.stars_seed is None:
```

Comment:
Wouldn't we want these things in the main pipeline as well? I guess I'm not sure why this test has turned into its own file & own development.

Reply:
Agreed. As we have discussed on Slack, we should not merge `mock_superBIT_gaussian.py` into master but instead work on a dedicated branch.